src/libs/ActiveClientManager/index.ts

```diff
@@ -4,25 +4,24 @@
  * This file ensures exactly that by tracking all the clientIDs connected, storing the most recent one last and it considers that last clientID the "leader".
  */
 
-import _ from 'underscore';
 import Onyx from 'react-native-onyx';
 import Str from 'expensify-common/lib/str';
 import ONYXKEYS from '../../ONYXKEYS';
 import * as ActiveClients from '../actions/ActiveClients';
+import ActiveClientManagerModule from './types';
 
 const clientID = Str.guid();
 const maxClients = 20;
-let activeClients = [];
-let resolveSavedSelfPromise;
-const savedSelfPromise = new Promise((resolve) => {
+let activeClients: string[] = [];
+let resolveSavedSelfPromise: () => void;
+const savedSelfPromise = new Promise<void>((resolve) => {
     resolveSavedSelfPromise = resolve;
 });
 
 /**
  * Determines when the client is ready. We need to wait both till we saved our ID in onyx AND the init method was called
- * @returns {Promise}
  */
-function isReady() {
+function isReady(): Promise<void> {
```

Comment:
It is one line function and TS sets type implicitly. 
```suggestion
function isReady() {
```